main.py

```diff
@@ -1,12 +1,171 @@
+            videos.delete_video(video_id)
+            print(f"Video {video_id} has been deleted")
+
+        # list all videos
+        elif choice=='4':
+            for video in videos.list_videos():
+                print(video)
+
+        # get one video
+        elif choice=='5':
+            video_id = input("Please enter an ID: ")
+            selected_video = videos.get_video(video_id=video_id)
+            print(selected_video)
+
+        # add a customer
+        elif choice=='6':
+            name=input("Please enter a name: ")
+            postal_code=input("Please enter a postal code: ")
+            phone=input("Please enter a phone number: ")
+            response = customers.add_customer(name=name, postal_code=postal_code,
+            phone=phone)
+            print(f"Customer {name} created")
+
+        # update a customer
+        elif choice=='7':
+            customer_id = input("Please enter an ID: ")
+            name=input("Please enter a new name: ")
+            postal_code=input("Please enter a new postal code: ")
+            phone=input("Please enter a new phone: ")
+
+            response = customers.update_customer(customer_id=customer_id,
+            name=name, postal_code=postal_code, phone=phone)
+            print(f"Updated customer id {customer_id}")
+
+        # delete a customer
+        elif choice=='8':
+            customer_id = input("Please enter an ID: ")
+            response = customers.delete_customer(customer_id)
+            print(f"Customer number {customer_id} has been deleted")
+
+        # get info about one customer
+        
+        elif choice=='9':
+            customer_id = input("Please enter an ID: ")
+            selected_customer = customers.get_one_customer(customer_id=customer_id)
+            if selected_customer:
+                print(selected_customer)
+
+        # get info about all customers
+        elif choice=='10':
+            for customers in customers.all_customers():
+                print(customers)
+
+        # check out video 
+        elif choice=='11':
+            customer_id=input("Please enter a customer id: ")
+            video_id=input("Please enter a video id: ")
+            response = rentals.check_out(customer_id=customer_id,
+            video_id=video_id)
+            print("Video successfully checked out")
+            print(f"Video id {video_id} checked out to {customer_id}")
+
+        # check in video 
+        elif choice=='12':
+            customer_id=input("Please enter a customer id: ")
+            video_id=input("Please enter a video id: ")
+            response = rentals.check_in(customer_id=customer_id,
+            video_id=video_id)
+            print("Video successfully checked in")
+            print(f"Video {video_id} checked in from {customer_id}")
+
+        
+        # end
+        elif choice=='00':
+            play=False
+            print("\n Thank you for visiting our store. Come again soon.")
+
+      
```

Comment:
Great job separating the customer, rental, and video requests from the menu logic!